<?xml version="1.0"?>
<robot name="test_tube" xmlns:xacro="http://www.ros.org/wiki/xacro">
  <xacro:arg name="prefix" default="test_tube_1_"/>

  <!-- Includo URDF e argomento -->
  <xacro:include filename="$(find YumiProjectThesis)/urdf/test_tube.urdf.xacro"/>
  <xacro:test_tube_urdf prefix="$(arg prefix)"/>

  <!-- Gazebo -->
  <xacro:include filename="$(find YumiProjectThesis)/urdf/test_tube.gazebo.xacro"/>
  <xacro:test_tube_gazebo prefix="$(arg prefix)"/>
</robot>
